apps/docs/src/guide/contracts/methods.md

```diff
@@ -24,12 +24,20 @@ A funded wallet it's required:
 
 ## `call`
 
-The `call` method should be used to submit a real contract call transaction to the node.
+The `call` method submits a real contract call transaction to the node. It resolves as soon as the transaction is submitted and returns a `transactionId` along with a `waitForResult` callback to wait for the transaction execution.
```

Comment:
```suggestion
The `call` method submits a real contract call transaction to the node. It resolves as soon as the transaction is submitted and returns a `transactionId` along with a `waitForResult` callback to wait for the transaction execution.

As is the natural behaviour of blockchains, transactions may take a few seconds before they are resolved on chain. Therefore this method is useful where you want to submit a transaction but do not want it's result to block further processing of your application.
```

Some reasoning / use-case I think would be beneficial here.

Reply:
I've modified this part https://github.com/FuelLabs/fuels-ts/commit/536e87ba3f46fc7ba84e3d0b360a5a4101819d76